payments-core/src/main/java/com/stripe/android/view/CardMultilineWidget.kt

```diff
@@ -393,13 +396,9 @@ class CardMultilineWidget @JvmOverloads constructor(
             cardNumberTextInputLayout.isLoading = it
         }
 
-        isEnabled = true
-    }
-
-    override fun onAttachedToWindow() {
-        super.onAttachedToWindow()
         postalCodeEditText.config = PostalCodeEditText.Config.Global
         cvcEditText.hint = null
```

Comment:
Not sure why this was in `onAttachedToWindow`, but it seems wrong since it'll overwrite any value set during `onCreate`, after the view was already created. Moved to the end of init so that doesn't happen.